trento/xml/article_sap_trento.xml

```diff
@@ -2517,21 +2517,101 @@ trento-server-rabbitmq-0</screen>
+               <listitem>
+          <para>Host clean-up capabilities, allowing users to get rid of hosts that are no longer part of their SAP environment.</para>
+        </listitem>
+       <listitem>
+          <para>New checks results view that leverages the potential of the new checks engine (wanda) and provides the user with insightful information about the 
+           check execution.</para>
+        </listitem>
+      </varlistentry>
+           <varlistentry>
+        <term>Version 2.0.0</term>
+       <para> Consisting of Helm chart 2.0.0, web 2.0.0, cchecks engine (wanda) 1.0.0 and agent 2.0.0. Released on 2023/04/26.</para>
         <listitem>
-          <para></para>
+          <para>A brand-new safer, faster, ligher SSH-less configuration checks engine (wanda) which not only opens the door to configuration checks for other HA scenarios (ASCS/ER, HANA scaleup
+           cost optimized, HANA scale-out) and other targes in the environment (hosts, HANA databases, NetWeaver instances), but also will allow customization of existing checks and
+          addtion of custom checks.</para>
+        </listitem>
+               <listitem>
+          <para>Addtion of VMware to the list of known platforms.</para>
+        </listitem>
+       <listitem>
+          <para>Versioned external APIs for both the web and the checks engine (wanda) components. The full list of available APIs can be found at 
+          https://www.trento-project.io/web/swaggerui/ and https://www.trento-project.io/wanda/swaggerui/ respectiverly.</para>
+        </listitem>
+      </varlistentry>
+     <varlistentry>
+        <term>Version 1.2.0</term>
+       <para> Consisting of Helm chart 1.2.0, web 1.2.0, runner 1.1.0 and agent 1.1.0. Released on 2022/11/04.</para>
+        <listitem>
+          <para>Configration checks for HANA scale-up peformance optimized two-node clusters on on-premise bare metal platforms, including KVM and Nutanix.</para>
+        </listitem>
+               <listitem>
+          <para>A dynamic dashboard that allows you to determine, at a glance, the overall health status of your SAP environment.</para>
+        </listitem>
+      </varlistentry>
+     <varlistentry>
+        <term>Version 1.1.0</term>
+       <para> Consisting of Helm chart 1.1.0, web 1.1.0, runner 1.0.1 and agent 1.1.0. Released on 2022/07/14.</para>
+        <listitem>
+          <para>Fix for major bug in the runner component that prevented the web component to load properly.</para>
+        </listitem>
+      </varlistentry>
+     <varlistentry>
+        <term>Version 1.0.0 (general availability)</term>
+       <para> Consisting of Helm chart 1.0.0, web 1.0.0, runner 1.0.0 and agent 1.0.0. Released on 2022/04/29.</para>
+              <listitem>
+          <para>Clean, simple web console designed with SAP Basis in mind and that reacts, in real time, to changes happening in the 
+           backend environment thanks to the event-driven technology behind it.</para>
```

Comment:
* Basis -> basis? If it's not a unique name, use lowercase.
* Split the long sentence in two.

```suggestion
          <para>Clean, simple web console designed with SAP basis in mind. It reacts in real time to changes happening in the 
           backend environment thanks to the event-driven technology behind it.</para>
```